dist/bearhug-angular.js

```diff
@@ -0,0 +1,207 @@
+(function(angular) {
+    bearhugAuthenticator.$inject = ["$http", "$injector", "$q", "bearhugStorage"];
+    bearhugInterceptor.$inject = ["$q", "$injector", "bearhugStorage"];
+    bearhugStorage.$inject = ["bearerUtils"];
+    interceptorFilter.$inject = ["$q"];
+    angular.module("talis.bearhug", []);
+    angular.module("talis.bearhug").factory("bearerUtils", bearerUtils);
+    function bearerUtils() {
+        var BEARER_REGEX = /^Bearer (.*)$/;
```

Comment:
Should we be checking to see if characters do exist? Currently `Bearer ` is acceptable. Should we be checking for `/^Bearer (.+)$/`? Or `/^Bearer ([a-zA-Z0-9\.]+)$/`